vyper/codegen/abi.py

```diff
@@ -438,3 +438,50 @@ def abi_decode(lll_node, src, pos=None):
                 ['seq', lll_ret]]]
 
     return lll_ret
+
+
+def _add_ofst(loc, ofst):
```

Comment:
Is that supposed to be `offset`?

Reply:
I think `ofst` is a pretty standard abbreviation for `offset`.

Reply:
sure, but it only saves like 2 letters lol. unnecessarily optimization.

Reply:
I guess it's a style thing. Variables with limited scope get shorter names. Is this really important?

Reply:
It's better readability :man_shrugging: